packages/components/src/reaction_icon_bubble/index.module.scss

```diff
@@ -0,0 +1,33 @@
+@import "nebenan-ui-kit";
```

Comment:
i think you wanted to start using `@import "~nebenan-ui-kit";`

Reply:
Yes, I started out with ~ but I think I had issues with rollup. It wasn't able to find the file. Didn't investigate any deeper though

Reply:
worth looking into, add a ticket please

Reply:
https://favro.com/organization/aee06c3fe6503c804f507dd9/22c630cc02b09f2d6f98b8b3?card=Goo-33170